apps/bitwarden-mailconfig.sh

```diff
@@ -0,0 +1,218 @@
+#!/bin/bash
+
+# T&M Hansson IT AB © - 2020, https://www.hanssonit.se/
+
+# shellcheck disable=2034,2059
+true
+# shellcheck source=lib.sh
+. <(curl -sL https://raw.githubusercontent.com/nextcloud/vm/master/lib.sh)
+
+# Check for errors + debug code and abort if something isn't right
+# 1 = ON
+# 0 = OFF
+DEBUG=0
+debug_mode
+
+# Must be root
+root_check
+
+# Check if Bitwarden is already installed
+print_text_in_color "$ICyan" "Checking if Bitwarden is already installed..."
+if is_docker_running
+then
+    if docker ps -a --format '{{.Names}}' | grep -Eq "bitwarden";
+    then
+        if [ ! -d "$BITWARDEN_HOME"/bwdata ]
+        then
+            msg_box "It seems like 'Bitwarden' isn't installed in the new place.\n\nYou cannot run this script."
+            exit 1
+        fi
+    else
+        msg_box "It seems like 'Bitwarden' isn't installed.\n\nYou cannot run this script."
+        exit 1
+    fi
+else
+    msg_box "It seems like 'Bitwarden' isn't installed.\n\nYou cannot run this script."
+    exit 1
+fi
+
+msg_box "This script lets you configure your mailserver settings for Bitwarden."
+if [[ "no" == $(ask_yes_or_no "Do you want to continue?") ]]
+then
+    exit
+fi
+
+# Insert globalSettings__mail__smtp__trustServer to global.override
+if ! grep -q "^globalSettings__mail__smtp__trustServer=" "$BITWARDEN_HOME"/bwdata/env/global.override.env
+then
+    echo "globalSettings__mail__smtp__trustServer=false" >> "$BITWARDEN_HOME"/bwdata/env/global.override.env
+fi
+
+# Enter Mailserver
+while true
+do
+    MAIL_SERVER=$(whiptail --inputbox "Please enter the Mailserver, that you want to use.\nE.g. smtp.mail.de\nIf you don't want to change the Mailserver, that is already configured inside the global.override.env-file, just leave the box empty." "$WT_HEIGHT" "$WT_WIDTH" 3>&1 1>&2 2>&3)
+    if [[ "no" == $(ask_yes_or_no "Are your sure about your answer?") ]]
+    then
+        msg_box "It seems like your weren't satisfied by the Mailserver you entered. Please try again."
+    else
+        break
+    fi
+done
+
+# Enter if you want to use ssl
+while true
+do
+    USE_SSL=$(whiptail --inputbox "Do you want to use SSL for your Mailserver? If yes: please type in 'yes'.\nPlease note: if your Mailserver only supports Starttls, you cannot use SSL - so please answer in this case here 'no'.\nIf you don't want to change the SSL-setting, that is already configured inside the global.override.env-file, just leave the box empty." "$WT_HEIGHT" "$WT_WIDTH" 3>&1 1>&2 2>&3)
+    if [[ "no" == $(ask_yes_or_no "Are your sure about your answer?") ]]
+    then
+        msg_box "It seems like your weren't satisfied by the answer you entered. Please try again."
+    else
+        if [ "$USE_SSL" = "yes" ]
+        then
+            DEFAULT_PORT=465
```

Comment:
I think we should add 587 which is TLS and most common in SMTP.

Reply:
Hm, but is it also default in most cases together with SSL?

Reply:
SSL is older than TLS, less secure. 

I think we should provide SSL and TLS (587)

Reply:
it looks like there is an additional setting for starttls:
globalSettings__mail__smtp__startTls=true
https://github.com/bitwarden/server/issues/451#issuecomment-481676589

Reply:
Yup, that's it!